cmake/VASTRegisterPlugin.cmake

```diff
@@ -160,7 +160,15 @@ function (VASTCompileFlatBuffers)
   add_library(${FBS_TARGET} INTERFACE)
 
   # Link our target against FlatBuffers.
-  find_package(Flatbuffers REQUIRED CONFIG)
+  find_package(Flatbuffers CONFIG)
+  # Ugly workaround for Flatbuffers 2.0.8, which briefly changed the
+  # project name to "FlatBuffers".
```

Comment:
The same workaround needs to be appended to `VAST_FIND_DEPENDENCY_LIST` right below.